compiler/one-cmds/how-to-use-one-commands.txt

```diff
@@ -152,6 +152,7 @@ Current transformation options are
 - disable_validation : This will turn off operator validations.
 - fold_add_v2 : This removes AddV2 operation which can be folded
 - fold_cast : This removes Cast operation which can be folded
+- fold_dwconv : This folds Depthwise Convolution operation which can be folded
```

Comment:
(optional) maybe you may want to swap with below item?

Reply:
Sorry, I've read this comment a bit late... Here's PR which fixes sorting order: #7619 :)